src/main/java/org/fundacionjala/coding/german/DecodeString.java

```diff
@@ -0,0 +1,177 @@
+package org.fundacionjala.coding.german;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by seus on 25/8/2017.
+ */
+public class DecodeString {
+    private String stringOCR;
+    private String stringOCROut;
+    private List<String> listt;
+    private List<Integer> listOCR;
+
+    /**
+     * this is comment of function.
+     *
+     * @param stringOCR **this is string for decode**
+     */
+    public DecodeString(String stringOCR) {
+        this.stringOCR = stringOCR;
+        listt = new ArrayList<>();
+        listOCR = new ArrayList<>();
+        listt.add(String.format("%s%s%s", " _ ",
+                "| |",
+                "|_|"));
+
+
+        listt.add(String.format("%s%s%s", "   ",
+                "  |",
+                "  |"));
+
+        listt.add(String.format("%s%s%s", " _ ",
+                " _|",
+                "|_ "));
+
+        listt.add(String.format("%s%s%s", " _ ",
+                " _|",
+                " _|"));
+
+        listt.add(String.format("%s%s%s", "   ",
+                "|_|",
+                "  |"));
+
+        listt.add(String.format("%s%s%s", " _ ",
+                "|_ ",
+                " _|"));
+
+        listt.add(String.format("%s%s%s", " _ ",
+                "|_ ",
+                "|_|"));
+
+        listt.add(String.format("%s%s%s", " _ ",
+                "  |",
+                "  |"));
+
+        listt.add(String.format("%s%s%s", " _ ",
+                "|_|",
+                "|_|"));
+
+        listt.add(String.format("%s%s%s", " _ ",
```

Comment:
This list should be static final
Because it will not change the values during the execution.